zerver/openapi/zulip.yaml

```diff
@@ -4343,27 +4267,37 @@ components:
             URL for the the user's avatar.  Will be `null` if the `client_gravatar`
             query parameter was set to `True` and the user's avatar is hosted by
             the Gravatar provider (i.e. the user has never uploaded an avatar).
+
+            **Changes**: New in Zulip 2.1.0.
+          example: "x"
```

Comment:
So, the example isn't a URL, and more importantly, the User object in general had a `avatar_url` for all endpoints other than this one well before Zulip 2.1.0.  Same for `avatar_version`.  We made need to solve the problem of creating an optional syntax only for a specific caller of these common components.